website/pages/index.html

```diff
@@ -11,11 +12,9 @@
     <meta property="og:description" content="Simple API, no dependencies – yet super-fast on all major browsers.">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0-beta/dist/katex.min.css" integrity="sha384-9tPv11A+glH/on/wEu99NVwDPwkMQESOocs/ZGXPoIiLE8MU/qkqUcZ3zzL+6DuH" crossorigin="anonymous">
     <script src="https://cdn.jsdelivr.net/npm/katex@0.10.0-beta/dist/katex.min.js" integrity="sha384-U8Vrjwb8fuHMt6ewaCy8uqeUXv4oitYACKdB0VziCerzt011iQ/0TqlSlv8MReCm" crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/normalize.css@8.0.0/normalize.css" integrity="sha256-7GAtDQ79wTEOjhBKf70uBQG7A5yyb+8rigu07atXWDY=" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700,700i" rel="stylesheet">
     <link href="https://use.fontawesome.com/efe646278d.css" type="text/css" rel="stylesheet">
     <link href="static/index.css" type="text/css" rel="stylesheet">
-    <script src="//use.typekit.net/ulc1wme.js"></script>
-    <script>try{Typekit.load();}catch(e){}</script>
   </head>
   <body>
     <div class="header"><div class="container">
```

Comment:
It’d be nice to have a link to KaTeX contributors. Or better, have a page of [all-contributors](https://github.com/kentcdodds/all-contributors/blob/master/README.md) in the documentation.

Reply:
Sounds cool! Maybe make this into a separate issue, or add it to the documentation checklist? Do you have an example from another project of what this might look like?

Reply:
@edemaine I was thinking one of:
* a link to https://github.com/Khan/KaTeX/graphs/contributors
* a documentation page ([example](https://babeljs.io/team))
* Contributors section in READEME ([example](https://github.com/kentcdodds/all-contributors/blob/master/README.md#contributors))